nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/state/reporting-tasks/reporting-tasks.reducer.ts

```diff
@@ -61,6 +62,15 @@ export const reportingTasksReducer = createReducer(
         error,
         status: 'error' as const
     })),
+    on(configureReportingTaskSuccess, (state, { response }) => {
+        return produce(state, (draftState) => {
+            const componentIndex: number = draftState.reportingTasks.findIndex((f: any) => response.id === f.id);
+            if (componentIndex > -1) {
+                draftState.reportingTasks[componentIndex] = response.reportingTask;
+            }
+            draftState.saving = false;
+        });
+    }),
     on(createReportingTask, deleteReportingTask, (state, { request }) => ({
```

Comment:
We should add `configureReportingTask` here

```suggestion
    on(createReportingTask, deleteReportingTask, configureReportingTask, (state, { request }) => ({
        ...state,
        saving: true
    })),
```

Reply:
Good catch.